### ISSUE TITLE
Wrong example in README?

### ISSUE THREAD

**Issue by DevAlien** [epoberezkin/ajv-i18n#26]
Hi, 
great work btw.

I tried to use this lib and noticed that in the README it is written the following:

```
if (!valid) {
    // array of error objects with Russian messages
    var errors = localize.ru(validate.errors);
    // string with all errors and data paths
    console.log(ajv.errorsText(errors, { separator: '\n' }));
}
```

but, checking the language files it looks like it modifies the referenced object and does not return anything.

I could provide a PR to fix the documentation to something like

```
if (!valid) {
    // array of error objects with Russian messages
    localize.ru(validate.errors);
    // string with all errors and data paths
    console.log(ajv.errorsText(validate.errors, { separator: '\n' }));
}
```

Or make sure that localize returns the errors.

**Comment by epoberezkin** [comment]
yes, thank you

**Comment by epoberezkin** [comment]
I think modifying and not returning is better to avoid expectation that the original errors remain intact and new objects are created.

**Comment by epoberezkin** [comment]
closed by #27
